tests/aws/services/stepfunctions/v2/services/test_ecs_task_service.py

```diff
@@ -0,0 +1,516 @@
+            platform_version=ecs.FargatePlatformVersion.VERSION1_4
+        )
+        task_def = ecs.FargateTaskDefinition(stack, "taskdef", cpu=256, memory_limit_mib=512)
+        task_def.add_container(
+            "maincontainer",
+            image=ecs.ContainerImage.from_registry("no_such_image"),
+            entry_point=["echo", "hello"],
+            essential=True,
+        )
+
+        # state machine setup
+        run_task = tasks.EcsRunTask(
+            stack,
+            "ecstask",
+            cluster=cluster,
+            launch_target=launch_target,  # noqa
+            task_definition=task_def,
+            integration_pattern=sfn.IntegrationPattern.RUN_JOB,
+        )
+        definition_body = sfn.DefinitionBody.from_chainable(run_task)
+        statemachine = sfn.StateMachine(stack, "statemachine", definition_body=definition_body)
+
+        # stack outputs
+        cdk.CfnOutput(stack, "TaskDefinitionArn", value=task_def.task_definition_arn)
+        cdk.CfnOutput(stack, "ClusterArn", value=cluster.cluster_arn)
+        cdk.CfnOutput(stack, "StateMachineArn", value=statemachine.state_machine_arn)
+        cdk.CfnOutput(stack, "ClusterName", value=cluster.cluster_name)
+        cdk.CfnOutput(stack, "StateMachineRoleArn", value=statemachine.role.role_arn)
+        cdk.CfnOutput(stack, "TaskDefinitionFamily", value=task_def.family)
+        cdk.CfnOutput(
+            stack, "TaskDefinitionContainerName", value=task_def.default_container.container_name
+        )
+
+        # provisioning
+        with infra.provisioner(skip_deployment=False, skip_teardown=False) as prov:
+            yield prov
+
+    @markers.aws.validated
+    @markers.snapshot.skip_snapshot_verify(
+        paths=[
+            "$..Attachments..Details",
+            "$..Attachments..Id",
+            "$..Attachments..Status",
+            "$..Attachments..Type",
+            "$..AvailabilityZone",
+            "$..ClusterArn",
+            "$..Connectivity",
+            "$..ConnectivityAt",
+            "$..Cpu",
+            "$..DesiredStatus",
+            "$..ExecutionStoppedAt",
+            "$..GpuIds",
+            "$..Group",
+            "$..HealthStatus",
+            "$..ImageDigest",
+            "$..InferenceAccelerators",
+            "$..LastStatus",
+            "$..ManagedAgents",
+            "$..Memory",
+            "$..NetworkInterfaces",
+            "$..Overrides.ContainerOverrides",
+            "$..Overrides.InferenceAcceleratorOverrides",
+            "$..PlatformFamily",
+            "$..PullStartedAt",
+            "$..PullStoppedAt",
+            "$..RuntimeId",
+            "$..SdkHttpMetadata",
+            "$..SdkResponseMetadata",
+            "$..StartedAt",
+            "$..StartedBy",
+            "$..StopCode",
+            "$..StoppedAt",
+            "$..StoppedReason",
+            "$..StoppingAt",
+            "$..TaskDefinitionArn",
+            "$..Version",
+            "$..parameters.Cluster",
+        ]
```

Comment:
nit: not required for this PR but would maybe make sense to re-use this list across the tests in the files